dynamic/Dockerfile

```diff
@@ -26,27 +42,16 @@ ENV LANG=C.UTF-8 \
     INSTANA_REPOSITORY_PROXY_PASSWORD="" \
     INSTANA_REPOSITORY_PROXY_USE_DNS=""
 
-RUN apt-get update && \
-    apt-get install -y gnupg2 ca-certificates curl && \
-    echo "deb [arch=amd64] https://_:${FTP_PROXY}@packages.instana.io/agent/deb generic main" > /etc/apt/sources.list.d/instana-agent.list && \
-    echo "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable" > /etc/apt/sources.list.d/docker.list && \
-    apt-key adv --fetch-keys "https://packages.instana.io/Instana.gpg" && \
-    apt-key adv --fetch-keys "https://download.docker.com/linux/ubuntu/gpg" && \
-    apt-get update && \
-    apt-get install -y instana-agent-dynamic inotify-tools gomplate docker-ce-cli containerd.io python-pip iptables && \
-    apt-get purge -y gnupg2 && \
+RUN apt-get install -y instana-agent-dynamic
+
+# Cleanup
+RUN apt-get purge -y gnupg2 && \
     apt-get autoremove -y && \
     rm -rf /etc/apt/sources.list.d/instana-agent.list && \
     rm -rf /var/lib/apt/lists/* && \
     apt-get clean
 
-ADD org.ops4j.pax.logging.cfg /root/
-ADD org.ops4j.pax.url.mvn.cfg /root/
-ADD configuration.yaml /root/
-ADD com.instana.agent.main.sender.Backend.cfg.tmpl /root/
-ADD com.instana.agent.main.config.UpdateManager.cfg.tmpl /root/
-ADD com.instana.agent.bootstrap.AgentBootstrap.cfg.tmpl /root/
-ADD mvn-settings.xml.tmpl /root/
+COPY org.ops4j.pax.logging.cfg org.ops4j.pax.url.mvn.cfg configuration.yaml com.instana.agent.main.sender.Backend.cfg.tmpl com.instana.agent.main.config.UpdateManager.cfg.tmpl com.instana.agent.bootstrap.AgentBootstrap.cfg.tmpl mvn-settings.xml.tmpl /root/
```

Comment:
I think it would be more clear to put every file on its own line, last line being the destination. Like so:
```
COPY org.ops4j.pax.logging.cfg \
  org.ops4j.pax.url.mvn.cfg \
  /root/
```
(does not contain all files so don't copy-paste directly)